x-pack/plugins/cloud_integrations/cloud_full_story/server/assets/fullstory_library.js

```diff
@@ -1,10 +1,3 @@
-/*
- * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
- * or more contributor license agreements. Licensed under the Elastic License
- * 2.0; you may not use this file except in compliance with the Elastic License
- * 2.0.
- */
-
 /* @notice
  * Portions of this code are licensed under the following license:
```

Comment:
This doesn't tell us which license fullstory is under.  `fs.js.LICENSE.txt` is fullstory's equivalent of our notice,  with information on their package's dependencies.  We  need to include  `[fullstory, ...fs.js.license.txt]` in our notice.

Reply:
works for me, thanks for following up.  I'll leave myself as pending review until the gainsight portion is resolved.